Simplify Razor bug reports.

NTaylorMullen opened the pull request:
- We now have better mechanisms for capturing information so no need to scare users off.

Pilchie reviewed (changes requested):
What's with the submodule update?

TanayParikh commented on .github/ISSUE_TEMPLATE/30_razor_tooling.md:
```diff
@@ -15,38 +15,16 @@ For other types of questions, consider using [StackOverflow](https://stackoverfl
 
-### Describe the bug
-A clear and concise description of what the bug is.
+Select the `Razor: Report a Razor Issue` command in VSCode (`ctrl + shift + p`) and follow the instructions to gather information necessary for us to investigate. Once done you can replace this content with contents that are put on your clipboard.
```
```suggestion
Select the `Razor: Report a Razor Issue` command in VSCode (<kbd>CTRL</kbd> + <kbd>SHIFT</kbd> + <kbd>P</kbd>) and follow the instructions to gather information necessary for us to investigate. Paste below:

---
<Paste Content Here>
---

```

TanayParikh commented on .github/ISSUE_TEMPLATE/30_razor_tooling.md:
```diff
@@ -15,38 +15,16 @@ For other types of questions, consider using [StackOverflow](https://stackoverfl
-1. Run Visual Studio with the [/Log](https://docs.microsoft.com/en-us/visualstudio/ide/reference/log-devenv-exe?view=vs-2019) command line switch
-2. Reproduce the issue
-3. Provide the logs located at `%Temp%\VisualStudio\LSP`
+How to report a problem in Visual Studio: https://docs.microsoft.com/en-us/visualstudio/ide/how-to-report-a-problem-with-visual-studio?view=vs-2019
```
```suggestion
How to report a problem in Visual Studio: https://docs.microsoft.com/en-us/visualstudio/ide/how-to-report-a-problem-with-visual-studio
```

so we don't have to maintain the link 😄

TanayParikh reviewed (commented):
LGTM aside from the submodules Kevin mentioned 😄

NTaylorMullen commented:
> What's with the submodule update?

lol, total mishap 😄

Commit:
Simplify Razor bug reports.

- We now have better mechanisms for capturing information so no need to scare users off.

Commit:
Address code review feedback
